html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me.html">About Me</a></li>
```

Comment:
Hi Timothy, when referring to the file itself, you don't need to include the path. For instance, if you are in the about-me.html file, your href attribute would just be "about-me.html".